insights/parsers/cups_ppd.py

```diff
@@ -0,0 +1,55 @@
+"""
+CupsPpd - files ``/etc/cups/ppd/*``
+===================================
+
+Parser to parse the content of files ``/etc/cups/ppd/*``
+"""
+
+from insights import Parser
+from insights import parser
+from insights.specs import Specs
+from insights.parsers import SkipException
+
+
+@parser(Specs.cups_ppd)
+class CupsPpd(Parser, dict):
+    """
+    Class to parse ``/etc/cups/ppd/*`` files.
+
+    Sample output for files::
+
+        *PPD-Adobe: "4.3"
+        *FormatVersion: "4.3"
+        *FileVersion: "2.2"
+        *LanguageVersion: English
+        *LanguageEncoding: ISOLatin1
+        *cupsFilter2: "application/vnd.cups-pdf application/pdf 10 -"
+        *cupsFilter2: "application/vnd.cups-postscript application/postscript 10 -"
+        *cupsFilter2: "application/vnd.cups-banner application/vnd.cups-banner 10 -"
+
+    Examples:
+        >>> type(cups_ppd)
+        <class 'insights.parsers.cups_ppd.CupsPpd'>
+        >>> cups_ppd["PCFileName"]
```

Comment:
Would you please take the examples in the `Examples` from the above `Sample output`? since I don't see the `PCFileName` from it.

Reply:
Updated